### DISCUSSION TITLE
Have users observed performance issues with the switch from 1.3. to 2.x?

### DISCUSSION THREAD

**Discussion by qalisto** [ransome1/sleek#627] [Polls]
[sleek underwent a complete rewrite from v1.3.1 to v2.0.0](https://github.com/ransome1/sleek/discussions/501), incorporating React MUI for the UI. This poll seeks to understand if this transition has affected the user experience in terms of **performance** and, if so, how significantly. Your feedback on the impact is valuable.

**Comment by ransome1** [comment]
@qalisto I would like to re-open this poll but with a focus on sleek in general since the switch to React. Is it ok if I highjack this one?

**Reply by qalisto** [reply]
Of course!
